Scripts/Runtime/Util/Random/RandomExtension.cs

```diff
@@ -0,0 +1,36 @@
+using System.Runtime.CompilerServices;
+using Unity.Jobs.LowLevel.Unsafe;
+using Unity.Mathematics;
+
+/// <summary>
+/// Extension methods for use with a <see cref="Random"/> instance.
+/// </summary>
+public static class RandomExtension
```

Comment:
Is it `Extension` or `Extensions`?

We seem to be using both? Doesn't matter to me, just clarifying.

Reply:
See other comment for discussion